pkg/trimaran/loadvariationriskbalancing/analysis.go

```diff
@@ -73,7 +73,7 @@ func (rs *resourceStats) computeScore(margin float64, sensitivity float64) float
 
 	// evaluate overall risk factor
 	risk := (mu + sigma) / 2
-	klog.V(6).Infof("mu=%f; sigma=%f; margin=%f; sensitivity=%f; risk=%f", mu, sigma, margin, sensitivity, risk)
+	klog.V(6).InfoS("mu=", "mu", mu, " sigma=", "s", sigma, " margin=", "margin", margin, " sensitivity=", "sensitivity", sensitivity, " risk=", "risk", risk)
```

Comment:
This is still outstanding.

- Capitalize the first word
- The first parameter is a general msg, and shouldn't come with %v